model/version_db.go

```diff
@@ -314,6 +314,27 @@ func VersionUpdateOne(query interface{}, update interface{}) error {
 	)
 }
 
+func ActivateVersions(versions []Version) error {
```

Comment:
since all we need from the Version object is the id, does it make sense to instead take in a []string of version ids?